utils/publish/pre-publish-check-ci.ts

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env -S node -r @swc-node/register
+
+/**
+ * Pre-publish checks to verify that our publish will go smoothly.
+ *
+ * NOTE: This script is run by changesets when the Release Manager is publishing
+ * a new version of the WB packages.
+ */
+import path from "path";
+
+import ancesdir from "ancesdir";
+import fg from "fast-glob";
+
+import {
+    checkPrivate,
+    checkEntrypoints,
+    checkSource,
+    checkTypes,
+    checkPublishConfig,
+} from "./pre-publish-utils";
+
+// Get the root folder; use the LICENSE file as our root anchor.
+const rootDir = ancesdir(__dirname, "LICENSE");
```

Comment:
**note:** Other than making it so all checks are run before exiting (this is copied from Perseus and tested), this is the biggest change that isn't in Perseus - using ancesdir to locate the root directory rather than hard-coding it. This way it'll work if the file gets moved around.